changelog.txt

```diff
@@ -1,123 +1,314 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+Highlights
+----------
+	- Remove acceleration as supported property.
```

Comment:
Please use one space (instead of one tab) for offsetting the dash, see the signac changelog for a reference. Also, I recommend that code should _never_ use tabs unless required.

https://github.com/glotzerlab/signac/blob/master/changelog.txt

Reply:
Fixed in commit 1b11f32

Reply:
Actually fixed in commit 0391c58